lnbits/app.py

```diff
@@ -111,8 +111,8 @@ async def shutdown():
 
     # wait a bit to allow them to finish, so that cleanup can run without problems
     await asyncio.sleep(0.1)
-    WALLET = get_wallet_class()
-    await WALLET.cleanup()
+    wallet_class = get_wallet_class()
```

Comment:
- the old version (`WALLET`)  follows the naming conventions for global variables: upsercase separated by underscores
- `wallet_class` is actually not a `class` but an instance of a class

Reply:
what if we rename it to `get_funding_source` and `funding_source` on a later PR?

Reply:
why on a later PR?

Reply:
done